vbfml/scripts/train.py

```diff
@@ -87,38 +78,69 @@ def setup(ctx, learning_rate: float, dropout: float):
         if re.match(dataset_labels["v_qcd_nlo_17"], dataset_info.name):
             dataset_info.n_events = 0.01 * dataset_info.n_events
 
+    # Object containing data for different models
+    # (set of features, dropout rate etc.)
+    # Loaded from the YML configuration file
+    ymlfile = "../config/model_params.yml"
+    db = ModelDB(ymlfile)
```

Comment:
Suggest different name for "ModelDB" since this is not really a data base. It's a configuration file parser.

Reply:
You're right, that class name is changed to `ModelConfiguration` in 88caa8d